### ISSUE TITLE
Window borders are thinner when the window cannot be resized

### ISSUE THREAD

**Issue by synthakii** [rozniak/xfce-winxp-tc#358]
Just making this issue in case this was something that went unnoticed.

Windows which cannot be resized have a ever so slightly thinner border (3px) compared to resizable windows (4px). To my knowledge xfwm4 doesn't seem to support this.

![Screenshot_2024-09-29_05-06-24](https://github.com/user-attachments/assets/253a95e9-2b73-40d4-93d4-b9034e0abe86)
![Screenshot_2024-09-29_05-19-07](https://github.com/user-attachments/assets/3facb9e3-c195-4c23-ae8f-08b686f84bef)

Also amazing work with this project with the extents it goes to have accuracy! 😃

**Comment by rozniak** [comment]
Yeah I know the borders vary in size depending on the type of window. At the moment the theme is a compromise to not have them too thin or too thick since XFWM4 only uses the one style for all windows.

Unfortunately this is kind of a Windows-specific thing so I'm not sure it would be possible to upstream a feature to support this in XFWM4, unlike the stretched theme bitmaps in #216 . Perhaps something to consider whenever XFWM4 gets Wayland support, if this project is to fork it one day.
